src/account/AccountItem.js

```diff
@@ -1,6 +1,6 @@
 /* @flow */
```

Comment:
> This resulted in too much automatically pushing and popping of screens, which doesn't look good.

Describe this in more detail. I think you might be referring to a behavior that is intended.

Reply:
Currently on clicking account item, we navigate to realm screen with auto fill realm url and tries to connect to it. So realm screen is pushed and within few seconds auth screen is pushed. 

On this PR, we are directly sending user to auth screen. Thus having less visual effects (sudden) which looks better. 

I guess we have discussed about this in India meet up :)